### DISCUSSION TITLE
Displaying a text

### DISCUSSION THREAD

**Discussion by Gluups** [community/community#24286] [New to GitHub]
<p>Hello everybody,</p>
<p>See how my log is displayed : </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/2aa221b14f88fc1ed6acd1ae96b191ec08c67da8.jpeg" data-download-href="/uploads/short-url/659pZ9D5UPyD2GvigjkmhbL9VWM.jpg?dl=1" title="gitlog.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/6567976/181083871-6a5f79a4-0838-4106-a0bb-70ce0bbc1f13.jpeg" alt="gitlog.jpg" data-base62-sha1="659pZ9D5UPyD2GvigjkmhbL9VWM" width="690" height="297" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/2aa221b14f88fc1ed6acd1ae96b191ec08c67da8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">gitlog.jpg</span><span class="informations">886×382 48.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I read there <a href="https://github.com/gitextensions/gitextensions/wiki/Git-Encoding" rel="nofollow noopener">https://github.com/gitextensions/gitextensions/wiki/Git-Encoding</a></p>
<p>that the solution exists, but is no more integrated in Git.</p>
<p>What is the matter ?</p>

**Comment by ppremk** [comment]
<p>Hi <a class="mention" href="/gluups">@gluups</a> thank you for reaching out. Looking at the screen shot and the information provided, I could make out that there is something related to encoding which you are facing. Could you please help provide more context on what is it you are trying to achieve so that we could seek out a right solution for the issue you are facing? :) </p>

**Comment by Gluups** [comment]
<p>Hello,</p>
<p>Well, my system is Windows 10 1803, 64 bits.</p>
<p>I tried CHCP 65001, CHCP 1252, CHCP 28591, with the fonts Consolas and Lucinda Console, in the properties of the console. In all cases the display is the same.</p>
<p>And I presume what I want to try is visible on the screen copy : execute the “git log” command.</p>

**Comment by ppremk** [comment, marked as answer]
<p>Hey <a class="mention" href="/gluups">@gluups</a> thanks for reverting quickly with more details. I had done some research to the problem you are facing and would suggest you to try the following in your terminal</p>
<pre><code>set LC_ALL=C.UTF-8
</code></pre>
<p>and then retry the git log command.</p>
<p>For some context and understanding the links below have some good explanation:</p>
<p><a href="https://stackoverflow.com/questions/41139067/git-log-output-encoding-issues-on-windows-10-command-prompt/41416262" rel="nofollow noopener">git log output encoding issues</a></p>
<p><a href="https://unix.stackexchange.com/questions/87745/what-does-lc-all-c-do" rel="nofollow noopener">what does LC_ALL=C do?</a></p>
<p>Hope this helps. Please let us know if it works out. Good luck!</p>
<p>~ppremk</p>

**Comment by Gluups** [comment]
<p>Jolly good !</p>
<p>Thank you.</p>
<p>The only difficulty, to make that permanent, is that some dialog boxes are higher than the screen, and I had to blindly target the OK button with the tab key.</p>
<p>I did it, and shall now have time to discover the reading you propose.</p>
<p>In case somebody needs it for Windows 10 : in the explorer, right click on the node of the machine, then properties.</p>
<p>Advanced system parameters. You arrive on the “System properties” dialog box ; I guess there is a better shortcut for it, that I do not know for Windows 10.</p>
<p>In the “Advanced System Parameters” tab, you have got an “Environment variables” button, that opens a dialog box of the same name. You have to pay attention that one part of this dialog box is for the current user, and the other one for all users (provided you are administrator). In the part you choose, clic on new, put “LC_ALL” in the first field of the new dialog box, and “C.UTF-8” in the second field (without quotation marks, I presume you guess ; to rather copy/paste from what you gave than from what I copied via the keyboard). Validate twice.</p>
